powerline/segments/common/bat.py

```diff
@@ -49,7 +49,7 @@ def _fetch_battery_info(pl):
 						continue
 					devices.append(devpath)
 					pl.debug('Using DBUS+UPower with {0}', devpath)
-				if devices:
+				if False and devices:
```

Comment:
And this is why we do code review.  This was how I triggered the testing of the other segment.  I think now is a good time to say I'm home sick and on a lot of benadryl so a bit of extra code scrutiny is probably in order.